### PR TITLE
Update requitites for source build(rust version)

### PR DESCRIPTION
**Pull request by djKooks**
Hello,
I just found that compile via source has been failed by issue here:
```
error[E0658]: the `#[non_exhaustive]` attribute is an experimental feature
  --> /Users/kwangin.jung/.cargo/registry/src/github.com-1ecc6299db9ec823/whoami-0.8.1/src/lib.rs:40:1
   |
40 | #[non_exhaustive]
   | ^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/44109

error[E0658]: the `#[non_exhaustive]` attribute is an experimental feature
   --> /Users/kwangin.jung/.cargo/registry/src/github.com-1ecc6299db9ec823/whoami-0.8.1/src/lib.rs:102:1
    |
102 | #[non_exhaustive]
    | ^^^^^^^^^^^^^^^^^
    |
    = note: for more information, see https://github.com/rust-lang/rust/issues/44109
```
Seems `#[non_exhaustive]` is a new feature on Rust 1.40.0, so added as pre-requitites.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,11 @@ curl -L https://install.meilisearch.com | sh
 
 If you have the Rust toolchain already installed on your local system, clone the repository and change it to your working directory.
 
+Pre-requitites:
+```
+- Rust 1.40.0
+```
+
```
Comment (MarinPostma) [inline README.md:85]: Hello! thank you for your PR! 
meili always compile against latest stable. What do you think about that instead?

```suggestion
If you have the latest stable Rust toolchain already installed on your local system, clone the repository and change it to your working directory.
```

Comment (djKooks) [reply README.md]: @MarinPostma updated~

**Comment by Kerollmops** [comment]
Hey @djKooks, could you squash the PR in only one commit and rebase it on master, please?

Thank you for this PR BTW :)

(I will not blame you if you create a new one already from master 😁)

**Comment by djKooks** [comment]
@Kerollmops thanks for review~please check.

**Comment by Kerollmops** [comment]
It does seem like you are not up-to-date with master, but please wait until I merge #646 before rebasing, I'll merge this PR after.

**Comment by djKooks** [comment]
@Kerollmops thanks~

**Comment by Kerollmops** [comment]
That's up-to-date, thank you!

**Comment by djKooks** [comment]
@Kerollmops okay~I'll do it ASAP

**Pull request merged**